vispy/plot/plotwidget.py

```diff
@@ -260,10 +266,11 @@ def mesh(self, vertices=None, faces=None, vertex_colors=None,
                 raise ValueError('vertices, faces, and fname must be None if '
                                  'fname is not None')
         else:
-            meshdata = MeshData(vertices, faces)
+            meshdata = MeshData(vertices, faces, face_colors=face_colors,
+                                vertex_colors=vertex_colors)
```

Comment:
Do you need to do this? It seems like the lines below (instantiation of the Mesh class) will already set these parameters.